nginx-app/spec/redirector_spec.rb

```diff
@@ -37,49 +37,29 @@
     end
   end
 
-  describe 'setup links to redirect' do
+  describe 'setup single url redirect' do
     before do
       node.set['redirector']['urls'] = {
         'john-doe.example.org' => {
-          '/foo' => 'http://example.org/bar',
-          '/' => 'http://example.com',
-          '/bar' => 'http://example.com/bar'
+          '/' => 'http://example.com'
```

Comment:
Neither, the test data is just misleading. The same `http://` is hardcoded in the template too, so it ends up both in the template and in the test being `http://http://`. things therefore match, test green.